### ISSUE TITLE
Can't render links with parentheses

### ISSUE THREAD

**Issue by ooker777** [adam-p/markdown-here#248]
For example, when I need to render this `[How to Type Vietnamese Language](http://yourvietnamese.com/language/type-vietnamese.html#telex)`, it shows: [How to Type Vietnamese Language](http://yourvietnamese.com/language/type-vietnamese.html#telex)

**Comment by adam-p** [comment]
Yes, this is a problem with the Markdown rendering library MDH uses: https://github.com/chjj/marked/issues/366

I'm planning to switch libraries soon, so hopefully this will be fixed then.

**Comment by adam-p** [comment]
There's a really clunk workaround: replace the closing parentheses inside the link with `&#41;`. 

So your example becomes:  
`[Telex](http://en.wikipedia.org/wiki/Telex_(input_method&#41;#Rules)`
